src/compiler/checker.ts

```diff
@@ -38897,6 +38897,17 @@ namespace ts {
                     });
                 }
             }
+
+            function copyExportedSymbols(source: SymbolTable, meaning: SymbolFlags): void {

```

Comment:
```suggestion
            function copyNonExportedSymbols(source: SymbolTable, meaning: SymbolFlags): void {
```

Reply:
Uh, this is only called for `sourceFileOrModuleDeclarationSymbol.exports`

Reply:
Ah, I understand the confusion based on the apparent disagreement between the name and the implementation of the function. The issue is that `a` and `b` are both in `exports`, but their local visibility is quite different:

```ts
export const a = 0;
export { a as b };
```

That does suggest some ideas for a better name 👍